src/components/Layouts/ReportLayout.less

```diff
@@ -21,6 +21,17 @@ h1 {
   text-transform: capitalize;
 }
 
+.not-bordered {
+  border: none !important;
+  padding: 0 !important;
+}
+
+.bordered {
```

Comment:
are you sure about making it global? i fear somehow it will get nested like `bordered` within `bordered` that can cause unexpected results

Reply:
done.